integration/kubernetes/k8s-expose-ip.bats

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+# Saves the ip addresses
+declare -a IP_ADDRESSES
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	deployment="hello-world"
+	service="my-service"
+	port="8080"
+	get_pod_config_dir
+}
+
+@test "Expose IP Address" {
+	wait_time=20
+	sleep_time=2
+
+	# Create deployment
+	kubectl create -f "${pod_config_dir}/deployment-expose-ip.yaml"
+
+	# Check deployment creation
+	cmd="kubectl wait --for=condition=Available deployment/${deployment}"
+	waitForProcess "$wait_time" "$sleep_time" "$cmd"
+
+	# Get deployment
+	kubectl get deployments ${deployment}
+
+	# Describe deployment
+	kubectl describe deployments ${deployment}
+
+	# Display information about ReplicaSet objects
+	kubectl get replicasets
+	kubectl describe replicasets
+
+	# Expose deployment
+	kubectl expose deployment/${deployment} --type=LoadBalancer --name=${service}
+
+	# Get service
+	kubectl get services ${service}
+	kubectl describe services ${service}
+
+	# Check pods are running
+	cmd="kubectl get pods -o jsonpath='{.items[*].status.phase}' | grep Running"
```

Comment:
@GabyCT You can instead use a `kubectl get pod "podname"` here to get the specific pod, instead on relying that there should be no other pods running. You can also query by label as well.

Reply:
@amshinde , as this is a deployment, if you use 
```
$ kubectl get pod "hello-world"
Error from server (NotFound): pods "hello-world" not found
```
The pods have the name `hello-world` with a random termination
```
$ kubectl get pods
NAME                           READY   STATUS    RESTARTS   AGE
hello-world-648779775d-h2nh9   1/1     Running   0          20m
hello-world-648779775d-rwnkl   1/1     Running   0          20m
```
That is the main reason, I am getting the names of the pods like the above